### PR TITLE
Feature #1692 - Separate option for logging Input Validation issues

### PR DESCRIPTION
**Pull request by netniV**
This commit adds a separate option for feature #1692 to enable logging Input Validation issues even when developer mode is turned off

### CODE REVIEW

**Comment by cigamit** [comment]
We should remove the developer mode then?

**Comment by netniV** [comment]
No, developer mode is used for the SQL debugging etc.  This allows page validation to be tested properly without having logs spammed by all that debugging information 👍

**Comment by cigamit** [comment]
Developer mode was specifically written for auditing request validation gaps.  It's different from Developer Debug which log's the sea.

File: include/global_settings.php [hunk 1]
```diff
@@ -259,7 +259,7 @@
 					'friendly_name' => __('Graph Syntax'),
 					'default' => ''
 					),
-				'developer_mode' => array(
+				'log_validation' => array(
```
Comment (cigamit) [inline include/global_settings.php]: You should just rip this out, don't forget the upgrade to rename if it's set.

Comment (netniV) [reply include/global_settings.php]: I was going to ask about that.  I noticed there was another part above that sets the same field name.

**Comment by cigamit** [comment]
I will handle the database upgrade change separately.

**Pull request merged**